libff/algebra/fields/field_utils.tcc

```diff
@@ -13,11 +13,81 @@
 #include <complex>
 #include <stdexcept>
 
+#include <libff/algebra/fields/bigint.hpp>
 #include <libff/common/double.hpp>
 #include <libff/common/utils.hpp>
 
 namespace libff {
 
+template<mp_size_t n>
+size_t field_get_digit(const bigint<n> &v, const size_t c, const size_t i)
```

Comment:
What about using the same argument name as in the hpp? Looking at this file is isolation, on easily wonders what the "one-letter" names represent.

Reply:
I think this wasn't mentioned in the hpp.  Added a decl and changed some of the names.

Reply:
Ah I thought it was declared as:
```
size_t field_get_digit(const bigint<n> &v, const size_t digit_size, const size_t digit_index)
```

Maybe i looked at the declaration of `field_get_signed_digit` instead..

> Added a decl and changed some of the names.

Thanks